cmd/upgrade/main.go

```diff
@@ -29,14 +31,32 @@ func main() {
 	kind := os.Args[3]
 	name := os.Args[4]
 	openebsNamespace := os.Args[5]
```

Comment:
It's required to have a check on no.of required arguments. If it is converting to `cli` we can ignore this comment